src/components/players/list/index.module.scss

```diff
@@ -47,41 +47,42 @@
   .imageLine {
     width: 100%;
     margin-top: 4px;
-    border-bottom: 2px solid transparent;
+    border-bottom: 2px solid $background-color-grey;
   }
 
   .detailsContainer {
     display: flex;
-    align-items: center;
+    align-items: flex-start;
```

Comment:
actually did it because the design shows that both the players name and the jersey number should be aligned to the top instead of centering them horizontally in the div